### PR TITLE
Design the referral program page & include graphics

### PR DESCRIPTION
**Pull request by paulproteus**
- Thanks to @neynah for the graphics & design review.

- This includes some minor refactoring of _about.scss to increase
  reusability of those styles onto other pages (specifically the
  referral program page now, and maybe other pages in the future).

This includes graphics from @neynah's pull request, so:

Close #1295

### CODE REVIEW

**Comment by paulproteus** [comment]
@kentonv ready for your review/merge.

But also if anyone else has feedback, particularly on my SASS style, I'd be very interested in that.

File: shell/client/_referrals.scss [hunk 1]
```diff
@@ -0,0 +1,91 @@
+.referrals {
+  @extend %padded-info-page;
+
+  .pageheader {
+    background-color: #6d5eb7;
+    color: white;
+
+    // Use DIV nesting so that we can have two different backgrounds.
+    .leftbackground {
+      background-image: url("/referral-graphics/referral-edge.svg");
```
Comment (paulproteus) [inline shell/client/_referrals.scss]: Just a comment, not a problem:

I created a new directory `shell/public/referral-graphics/` and put them all in there.

File: shell/shared/shell.js [hunk 2]
```diff
@@ -386,6 +386,13 @@ if (Meteor.isClient) {
     }
   });
 
+  Template.referrals.helpers({
+    setDocumentTitle: function () {
```
Comment (paulproteus) [inline shell/shared/shell.js]: Not a problem, just a remark:

I had forgotten this last time. Thanks to (I believe it was) @zarvox for noting it.

File: shell/client/shell.html [hunk 3]
```diff
@@ -1158,97 +1158,105 @@ <h3 id="app-stats">Apps</h3>
...
-    <h2>Sharing is referring</h2>
-
-    <ol>
-      <li>You share a grain with your friend (who isn't already an Oasis user).</li>
-      <li>Your friend signs in.</li>
-      <li>Your friend shares a grain with any other person (including you).</li>
-    </ol>
+  <div class="referrals">
+    <h1>Referral Program</h1>
+    <div class="pageheader">
```
Comment (kentonv) [inline shell/client/shell.html]: `<header>`?

File: shell/client/shell.html [hunk 4]
```diff
@@ -1158,97 +1158,105 @@ <h3 id="app-stats">Apps</h3>
...
+      <div class="leftbackground">
```
Comment (kentonv) [inline shell/client/shell.html]: These nested divs don't look like they're accomplishing anything. You can set the two backgrounds on the "pageheader" element and remove "leftbackground", "rightbackground", and "stoprotating".

Comment (kentonv) [inline shell/client/shell.html]: You can specify multiple backgrounds like this:

    background: left/contain no-repeat url("/referral-graphics/referral-edge.svg"),
                right/contain no-repeat url("/referral-graphics/referral-edge-rotated.svg");

Comment (paulproteus) [inline shell/client/shell.html]: "But it's CSS3-only!"

http://caniuse.com/#feat=multibackgrounds says it's safe for us to use this, so I can adjust accordingly.

File: shell/client/shell.html (hunk 3 above)
Comment (paulproteus) [inline shell/client/shell.html]: Ooh fancy. I can do that.

**Comment by paulproteus** [comment]
Will address this today; thanks for review!

**Comment by paulproteus** [comment]
@kentonv ready for your re-review/merge!

**Comment by paulproteus** [comment]
FWIW @neynah helped me understand how I was supposed to align the top H2, so this changes that.

**Comment by kentonv** [comment]
Now that the cascading divs aren't necessary can you remove them from the template?

**Comment by paulproteus** [comment]
Thought I did; let me see

>

**Comment by paulproteus** [comment]
Actually removed the unnecessarily DIVs now!

**Pull request merged**